### DISCUSSION TITLE
Moment Reference Center

### DISCUSSION THREAD

**Discussion by fernandafenelon** [JSBSim-Team/jsbsim#569] [Q&A]
Hello everyone,

What is my "Moment Reference Center" for aerodynamic moments?
I understand that I have to provide the aerodynamic data considering that the forces are applied at "AERORP" and the aerodynamics moments are provideded around this same "AERORP" point.
This "AERORP" point is defined in xml.

Also, the JSBSim will automatically calculate an increment of moment due to the distance between CG and AERORP and the aerodynamic forces to output as:
'moments/l-aero-lbsft' 
'moments/m-aero-lbsft'
'moments/n-aero-lbsft'

Is this correct?

**Comment by seanmcleod** [comment]
@fernandafenelon where are you getting your aerodynamic data from? The source of your data needs to indicate what the MRP (Moment Reference Point), MRC (Moment Reference Center), AERORP, i.e. a number of slightly different terms for the same thing, is for the data you're planning on using.

> **Non-dimensional Coefficients**
> ... convenient to represent the resulting force on the body in terms of its force components and a moment about a **fixed arbitrary point**

Since the MRP can be an arbitrary point the supplier of the aerodynamic data needs to specify it as part of the data package. 

JSBSim does increment the total moment given the current, remember the cg will be moving during flight as fuel is burned etc., cg and it's distance to the AERORP. Here is the code that does that:

https://github.com/JSBSim-Team/jsbsim/blob/ddd025ae2ed76631b4a9b00cc2871a78c251d476/src/models/FGAerodynamics.cpp#L280

P.S
@fernandafenelon could you please mark any of your previous discussion questions as 'Answered' if they have been answered satisfactorily. This will help other JSBSim users in future to get an idea whether the responses given answered your question or not.

**Reply by fernandafenelon** [reply]
OK, thank you. 

I just wanted to be sure that AERORP was the right place to indicate my fixed arbitrary point (about which the aerodynamic data are provided.)

**Comment by Jemonde** [comment]
Hi ! I have just few question to set my ideas for good... 
 
So, said differently, IF my experimental coefficients (force, moments...) are expressed  in the Wind Frame with a MRP defined at the CG (considering the structural frame)  : the right thing to do is to define the AERORP and the CG at the same location ? 

Also, my plane does not experience CG variation due to fuel consumption ... BUT, if I want to simulate a variation of CG due to an evolution of the plane, without having to re-calculate my coefficients, I only have to change the location of CG  in the XML file (this will create a delta between CG and AERORP) and it will work fine ?

**Comment by seanmcleod** [comment]
> IF my experimental coefficients (force, moments...) are expressed in the Wind Frame with a MRP defined at the CG (considering the structural frame) : the right thing to do is to define the AERORP and the CG at the same location ?

Yes.

> Also, my plane does not experience CG variation due to fuel consumption ... BUT, if I want to simulate a variation of CG due to an evolution of the plane, without having to re-calculate my coefficients, I only have to change the location of CG in the XML file (this will create a delta between CG and AERORP) and it will work fine ?

Yes, assuming the evolution only really does affect the cg, what about the moments of inertia? And doesn't also change the aerodynamics.

**Reply by Jemonde** [reply]
Ok, thank you for your confirmation ! it helps 

You're right, it is likely that this will change inertias and aerodynamics ...  (maybe in  a first order attempt to make things right, I would multiply the Intertial MAtrix by the factor of the masses) , But for now i wanted to focus on the definition of references...
